docs/HTML.md

```diff
@@ -18,14 +18,15 @@ HTML rendering require the use of `fpdf.HTMLMixin`,
 that provides a new `write_html` method:
 
 ```python
-from fpdf import FPDF, HTMLMixin
+from fpdf import FPDF
```

Comment:
It's nice that you removed those reference to `HTMLMixin` in the documentation.
There are other occurences though, in `DocumentOutlineAndTableOfContents.md`, `Links.md`, `Tables.md` & `Text.md`